assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -209,10 +209,14 @@ che.docker.ip.external=NULL
 #   - 'default':      internal address is address of docker host and ephemeral port are used
 #   - 'docker-local': internal address is address of container within docker network, and exposed ports
 #                     are used.
+#   - 'custom': The evaluation strategy may be customized through a template property.
 # The 'docker-local' strategy may be useful if a firewall prevents communication between che-server and
 # workspace containers, but will prevent communication when che-server and workspace containers are not
 # on the same Docker network.
-che.docker.server_evaluation_strategy=default
+che.docker.server_evaluation_strategy=custom
```

Comment:
Looks like you change default behavior of Che to work behind concrete proxy. 
It is really backward incompatible change. I think we should consider it as an additional feature instead of mandatory part of Che.
Some companies might want to embed Che into their system and don't have to use that proxy.